src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -82,25 +82,25 @@ public ExcessHet() {
     static Pair<Integer, Double> calculateEH(final VariantContext vc, final GenotypesContext genotypes) {
         final GenotypeCounts t = GenotypeUtils.computeDiploidGenotypeCounts(vc, genotypes, ROUND_GENOTYPE_COUNTS);
         // number of samples that have likelihoods
-        final int sampleCount = (int) genotypes.stream().filter(g->GenotypeUtils.isDiploidWithLikelihoods(g)).count();
+        final int sampleCount = (int) genotypes.stream().filter(GenotypeUtils::isDiploidWithLikelihoods).count();
 
-        return calculateEH(vc, t, sampleCount);
+        return calculateEH(t, sampleCount);
     }
 
     @VisibleForTesting
-    public static Pair<Integer, Double> calculateEH(final VariantContext vc, final GenotypeCounts t, final int sampleCount) {
```

Comment:
Not sure why this unused argument was here, but I took it out.

Reply:
I may have done a sloppy refactor -- it looks like I'm the one who last changed the method signature.  I'm not worried about this, but thanks for the cleanup.